apps/storybook/src/components/ai-conversation-history/ai-conversation-history.stories.tsx

```
@@ -0,0 +1,114 @@
+import { useState } from "react";
+import type { Meta, StoryObj } from "@storybook/react";
```

ESLint

[error] Do not import renderer package "`@storybook/react`" directly. Use a framework package instead (e.g. `@storybook/nextjs`, `@storybook/react-vite`, `@storybook/react-webpack5`, `@storybook/react-native-web-vite`, `@storybook/experimental-nextjs-vite`).

(storybook/no-renderer-packages)